src/test/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderMatchAPIIntegrationTest.kt

```diff
@@ -70,17 +79,77 @@ internal class OffenderMatchControllerAPIIntegrationTest : AbstractTestExecution
 
   @Test
   internal fun `should match when a single offender has all matching attributes`() {
+    loadOffenders(
+        OffenderIdentification(
+            surname = "gramsci",
+            firstName = "anne",
+            dateOfBirth = LocalDate.of(1988, 1, 6),
+            crn = "X00007",
+            nomsNumber = "G5555TT",
+            croNumber = "SF80/655108T",
+            pncNumber = "2018/x"
+        ),
+        OffenderIdentification(
+            surname = "smith",
+            firstName = "john",
+            dateOfBirth = LocalDate.of(1921, 1, 6),
+            crn = "X00001"
+        )
+    )
+
     given()
         .auth()
         .oauth2(jwtAuthenticationHelper.createJwt("ROLE_COMMUNITY"))
         .contentType(MediaType.APPLICATION_JSON_VALUE)
-        .body(MatchRequest(surname = "gramsci", firstName = "anne", dateOfBirth = LocalDate.of(1988, 1, 6)))
+        .body(MatchRequest(
+            surname = "gramsci",
+            firstName = "anne",
+            dateOfBirth = LocalDate.of(1988, 1, 6),
+            nomsNumber = "G5555TT",
+            croNumber = "SF80/655108T",
+            pncNumber = "2018/0123456"
+        ))
         .post("/match")
         .then()
         .statusCode(200)
         .body("matches.findall.size()", equalTo(1))
         .body("matches[0].offender.otherIds.crn", equalTo("X00007"))
   }
 
+  private fun loadOffenders(vararg offenders: OffenderIdentification) {
+    val template = "/elasticsearchdata/offender-template.json".readResourceAsText()
+    val templateOffender = objectMapper.readValue(template, OffenderDetail::class.java)
+
+    val offendersToLoad = offenders.map {
+      templateOffender.copy(
```

Comment:
I like this Kotlin `copy` malarkey - clone and amend an object at the same time - never need `toBuilder()` again :-)